src/modules/hyprland/workspaces.cpp

```diff
@@ -97,7 +97,7 @@ void Workspaces::onEvent(const std::string &ev) {
     for (Json::Value workspace_json : workspaces_json) {
       if (workspace_json["name"].asString() == payload &&
           (all_outputs() || bar_.output->name == workspace_json["monitor"].asString()) &&
-          (show_special() || workspace_json["name"].asString().find("special:") != 0)) {
+          (show_special() || workspace_json["name"].asString().find("special") != 0)) {
```

Comment:
You can change it to starts_with now that https://github.com/Alexays/Waybar/pull/2322 is merged. I'll also apply this fix in my PR.

Reply:
I just want to get this fixed asap, since it's showing the special workspace if it's open before the bar starts which I don't like.
feel free to change all the occurences